test/e2e/e2e_test.go

```diff
@@ -111,7 +111,55 @@ func TestMultiNodeCatalyst(t *testing.T) {
 	requireStreamRedirection(t, c1, c2)
 }
 
-func TestMultiNodeCatalyst_BootstrapNode(t *testing.T) {
+func TestStreamInterruption(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping testing in short mode")
+	}
+
+	// given
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	network := createNetwork(ctx, t)
+	defer network.Remove(ctx)
+
+	h1 := randomString("catalyst-")
+	h2 := randomString("catalyst-")
+
+	// when
+	c1 := startCatalyst(ctx, t, h1, network.name, defaultMistConfig())
+	defer c1.Terminate(ctx)
+	c2 := startCatalyst(ctx, t, h2, network.name, mistConfigConnectTo(h1))
+	defer c2.Terminate(ctx)
+
+	// then
+	requireMembersJoined(t, c1, c2)
+
+	// Begin ingesting and verify replication is working at catalyst-two node
+	p1 := startStream(t, c1)
+	requireReplicatedStream(t, c2)
+
+	// Stop and then restart the ingest stream
+	p1.Kill()
+	p2 := startStream(t, c1)
+
+	// Start load test to simulate multiple viewers at catalyst-two node
+	// TODO: This test will succeed until issue #65 is resolved. The current
```

Comment:
#65 is resolved already. Then, from what I understand, this test will fail. In that case, I suggest to not merge it right now.

Reply:
Sorry, meant issue 65 in mistserver repo: 
https://github.com/DDVTECH/mistserver/issues/65
I'll clarify in the comment. Also, the test is forced to pass even with a failure so that it will actually fail when the issue is fixed and we'll address the test correctly at that time.